restore-sql-server-transaction-logs/scheduled-upload/upload-script.ps1

```diff
@@ -0,0 +1,119 @@
+# Default variables
```

Comment:
In the README, you may want to explain why you didn't pick the same language for both the upload and the processing part.

Reply:
Done.

Reply:
LGTM